examples/rest/crypto-snapshots_all_tickers.py

```diff
@@ -0,0 +1,45 @@
+from polygon import RESTClient
+from polygon.rest.models import (
+    TickerSnapshot,
+    Agg,
+)
+
+# docs
+# https://polygon.io/docs/crypto/get_v2_snapshot_locale_global_markets_crypto_tickers
+# https://polygon-api-client.readthedocs.io/en/latest/Snapshot.html#get-all-snapshots
+
+# client = RESTClient("XXXXXX") # hardcoded api_key is used
+client = RESTClient()  # POLYGON_API_KEY environment variable is used
+
+snapshot = client.get_snapshot_all("crypto")  # all tickers
+
+# print raw values
+print(snapshot)
+
+# crunch some numbers
+for item in snapshot:
+
+    # verify this is an TickerSnapshot
+    if isinstance(item, TickerSnapshot):
+
+        # verify this is an Agg
+        if isinstance(item.prev_day, Agg):
+
+            # verify this is a float
+            if isinstance(item.prev_day.open, float) and isinstance(
+                item.prev_day.close, float
+            ):
+
+                percent_change = (
+                    (item.prev_day.close - item.prev_day.open)
+                    / item.prev_day.open
+                    * 100
+                )
```

Comment:
Tbh I'm not the biggest fan of this style. If you did this to satisfy a max line number count then I get it. But overall this function is really tall with all the extra whitespace and type checking

Reply:
Yeah, I'm not sure why I did that. I'll fix it. Makes sense and agree with you.

Reply:
Fixed

Reply:
Tagged you in the comment there. Makes sense now that I didn't actually do that and it was the `poetry run black polygon test_* examples` command that's actually reformatting this stuff so the lint checks pass. Not sure what the options are here.